js/app/eu-cookie-module.js

```diff
@@ -0,0 +1,33 @@
+//Show hide EU Cookie settings.
+(function ($) {
+  Drupal.behaviors.ucl_indigo_eu_cookie_module = {
+    attach: function (context, settings) {
+      // Show hide EU Cookie description settings.
+      function cookieShowHide() {
+        $("#eu-cookie-compliance-settings_show-hide").toggle();
+      }
+
+      // Show hide individual EU Cookie descriptions.
+      function cookieDescShowHide() {
+        var para = $(this).siblings(".eu-cookie-compliance-category-description").first();
+        $(para).toggle();
+      }
+      function initCookieSettings() {
+        if ($("#eu-cookie-compliance-settings_show-hide").css("display") !== "none") {
+          $("#eu-cookie-compliance-settings_show-hide").css("display", "none");
+        }
+        if ($(".eu-cookie-compliance-category-description").css("display") !== "none") {
+          $(".eu-cookie-compliance-category-description").css("display", "none");
+        }
+      }
+
+      $(document).ready(function() {
```

Comment:
This is wrapped inside drupal.behaviours so this isn't required - see https://www.drupal.org/node/756722

Reply:
I've removed ` $(document).ready(function() {  })` and it breaks without it